docs/_includes/ds6/listbox.html

```diff
@@ -0,0 +1,185 @@
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="listbox">
+                <button class="listbox__button expand-btn" aria-haspopup="listbox1" type="button">
+                    <span class="expand-btn__cell">
+                        <span>Listbox</span>
+                        <span class="expand-btn__icon"></span>
+                    </span>
+                </button>
+                <div id="listbox1" class="listbox__options" role="listbox">
+                    <div class="listbox__option" role="option">
+                        <span>Item 1</span>
+                    </div>
+                    <div class="listbox__option" role="option">
+                        <span>Item 2</span>
+                    </div>
+                    <div class="listbox__option" role="option">
+                        <span>Item 3</span>
+                    </div>
+                </div>
+                <select name="nativeOptions" hidden>
+                    <option value="item1">Item 1</option>
+                    <option value="item2">Item 2</option>
+                    <option value="item3">Item 3</option>
+                </select>
+            </span>
+        </div>
+        {% highlight html %}
+<span class="listbox">
+    <button class="listbox__button expand-btn" aria-haspopup="listbox1" type="button">
+        <span class="expand-btn__cell">
+            <span>Listbox</span>
+            <span class="expand-btn__icon"></span>
+        </span>
+    </button>
+    <div id="listbox1" class="listbox__options" role="listbox">
+        <div class="listbox__option" role="option">
+            <span>Item 1</span>
+        </div>
+        <div class="listbox__option" role="option">
+            <span>Item 2</span>
+        </div>
+        <div class="listbox__option" role="option">
+            <span>Item 3</span>
+        </div>
+    </div>
+    <select name="nativeOptions" hidden>
+        <option value="item1">Item 1</option>
+        <option value="item2">Item 2</option>
+        <option value="item3">Item 3</option>
+    </select>
+</span>
+        {% endhighlight %}
+    </div>
+
+    <h3>Selected Listbox Option</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="listbox">
+                <button class="listbox__button expand-btn" aria-haspopup="listbox1" type="button">
+                    <span class="expand-btn__cell">
+                        <span>Listbox</span>
+                        <span class="expand-btn__icon"></span>
+                    </span>
+                </button>
+                <div id="listbox1" class="listbox__options" role="listbox">
+                    <div class="listbox__option" role="option">
+                        <span>Item 1</span>
+                        <span class="listbox__status"></span>
+                    </div>
+                    <div class="listbox__option" role="option" selected>
```

Comment:
What is this `selected` for?

Reply:
Oops. I need to fix that. It needs to be `aria-selected`.